tests/dft-smoke/input.dat

```diff
@@ -16,7 +16,7 @@ H 1 1.0
 H 1 1.0 2 104.5
 }
 
-
+set dft_weights_tolerance -100.0
```

Comment:
Hey, that's cheating! :stuck_out_tongue_winking_eye:

Reply:
"This is not the line you are looking for"